SecurityPkg/SecurityPkg.dec

```diff
@@ -230,6 +230,9 @@
   ## GUID used to generate Spdm Uid
   gEfiDeviceSecuritySpdmUidGuid = {0xe37b5665, 0x5ef9, 0x4e7e, {0xb4, 0x91, 0xd6, 0x78, 0xab, 0xff, 0xfb, 0xcb }}
 
+  ## HOB contains TPM instance which is the same as PcdTpmInstanceGuid.

```

Comment:
GUID used to tag the HOB indicating the TPM instance.
The GUIDed HOB contains the same value as PcdGetPtr (PcdTpmInstanceGuid).

Reply:
Changed in latest code